distribution/sdk/sample-nsili-client/src/main/java/org/codice/alliance/nsili/client/Client.java

```diff
@@ -118,6 +85,43 @@ public void runTests(String iorUrl, String emailAddress) throws Exception {
     System.exit(0);
   }
 
+  private void evaluateResult(SampleNsiliClient sampleNsiliClient, DAG[] results, int index)
+      throws Exception {
+    LOGGER.info("\t RESULT : {} of {} ", (index + 1), results.length);
+    String obsoleteName = NsiliCardStatus.OBSOLETE.name();
+    if (getAttributeFromDag(results[index], NsiliConstants.STATUS).equalsIgnoreCase(obsoleteName)) {
+      LOGGER.info("Record is {}. Not testing result", obsoleteName);
+    } else {
+      sampleNsiliClient.printDagAttributes(results[index]);
+      if (SHOULD_DOWNLOAD_PRODUCT) {
+        sampleNsiliClient.downloadProductFromDAG(results[index]);
+      }
+
+      // ProductMgr
+      LOGGER.info("-----------------------");
+      try {
+        sampleNsiliClient.testProductMgr(results[index]);
+      } catch (Exception e) {
+        LOGGER.info("Unable to test ProductMgr: {}", NsilCorbaExceptionUtil.getExceptionDetails(e));
+      }
+      LOGGER.info("-----------------------");
+
+      // OrderMgr
+      PackageElement[] packageElements = sampleNsiliClient.order(results[index]);
+
+      // ProductMgr
+      if (SHOULD_PROCESS_PKG_ELEMENTS) {
+        // TODO prod is null for packageElement https://codice.atlassian.net/browse/CAL-192
```

Comment:
It looks like the issue referenced here is completed. Maybe this comment can be removed? @emmberk

Reply:
I don't remember why I added this comment. It looks like I noted that `packageElement.prod` will always be `null`. The comment can probably just be removed.

Reply:
Removed